src/components/button.js

```diff
@@ -0,0 +1,17 @@
+import React, { useState } from 'react'
+import {styles} from './input'
+import Input, {input} from './input'
+
+
+function Button (props) {
+    const [submit, setSubmit] = useState('')
```

Comment:
rather than controlling the submit state inside the button could we just pass the onPress as a prop?